src/utils/track.ts

```diff
@@ -4,20 +4,18 @@ let AWSCShortbread;
 let s;
 let AWSMA;
 
-if (typeof window !== "undefined") {
+if (typeof window !== "undefined" && typeof window.AWSCShortbread !== "undefined") {
```

Comment:
Have we verified that the `window.AWSCShortbread` is available when this code runs? What if the script hasn't loaded when this script runs? Do we need to listen to an event or something?

Reply:
We are just using script tag order to ensure that the shortbread scripts are running first, if we are able to async/defer these scripts later down the line we'll have to listen to an event here but for now script ordering will work

Reply:
Makes sense, thanks for the explanation